apps/trustlab/src/payload/access/abilities.js

```diff
@@ -0,0 +1,67 @@
+import { checkRole } from "./checkRole";
+import { ROLE_ADMIN, ROLE_AUTHOR, ROLE_DEFAULT, ROLE_EDITOR } from "./roles";
+
+export const canManageContent = (user) => {
+  if (!user) return false;
+  const isAdmin = checkRole([ROLE_ADMIN], user);
+  if (isAdmin) return true;
+
+  const isEditor = checkRole([ROLE_EDITOR], user);
+  if (isEditor)
+    return {
+      or: [
+        {
+          createdBy: {
+            equals: user.id,
+          },
+        },
+        {
+          "createdBy.role": {
+            in: [ROLE_EDITOR, ROLE_AUTHOR],
+          },
+        },
+      ],
+    };
```

Comment:
Editors should be able to edit any content on the site. Whether the content was created by themselves, authors, other editors or admins, it shouldn't matter.